deepchem/models/tensorgraph/tests/test_layers_eager.py

```diff
@@ -0,0 +1,980 @@
+import deepchem as dc
+import numpy as np
+import tensorflow as tf
+import deepchem.models.tensorgraph.layers as layers
+import tensorflow.contrib.eager as tfe
+from tensorflow.python.eager import context
+from tensorflow.python.framework import test_util
+import rdkit
+
+
+class TestLayersEager(test_util.TensorFlowTestCase):
+  """
+  Test that layers function in eager mode.
+  """
+
+  def test_conv_1d(self):
+    """Test invoking Conv1D in eager mode."""
+    with context.eager_mode():
```

Comment:
nit: can only have one level of indentation with multiple with syntax.

Reply:
I'm not sure what you mean?  The code works correctly.  The indentation is set by yapf.  Both "with" statements are required for it to work correctly.

Reply:
It is a python language construct that I didn't know about until recently.  I'm not saying we have to use it or that we even should use it I just thought I could share.

https://stackoverflow.com/questions/893333/multiple-variables-in-python-with-statement
```python
with context.eager_mode() as _, tfe.IsolateTest() as _:
  # All The Code
```

Reply:
Cool, I didn't know about that feature.